src/main/java/Dawn/Deadline.java

```diff
@@ -0,0 +1,47 @@
+package Dawn;
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+public class Deadline extends Task {
+    String deadline;
+    LocalDate date;
```

Comment:
should declare access modifier to make it clear is this variable are intended to be accessible by other classes or packages and what parts are meant to be internal. You may declare it as `private` attribute as you have a getter for the deadline.